test/unit/lib/censorDeath.js

```diff
@@ -0,0 +1,268 @@
+'use strict';
+const censorDeath = require('../../../src/lib/censorDeath');
+const roleWithFullAccess = ['full-details'];
+const roleWithoutFullAccess = ['births'];
+
+const deathData = {
+  id: 1000,
+  date: "2020-09-23",
+  entryNumber: 0,
+  inquestDate: "2020-09-23",
+  deceased: {
+    prefix: "Mr",
+    forenames: "Forename1 Forname2",
+    surname: "SURNAME",
+    suffix: 'Esquire',
+    dateOfBirth: "2020-09-23",
+    dateOfDeath: "2020-09-23",
+    dateOfDeathQualifier: "On or about",
+    ageAtDeath: 20,
+    birthplace: "Birth Address",
+    deathplace: "Death Address",
+    sex: "Indeterminate",
+    address: "Current Address",
+    occupation: "Rock God",
+    retired: false,
+    maidenSurname: null,
+    causeOfDeath: "Natural causes",
+    certifiedBy: "Certification",
+    relationshipToPartner: "Bit on side",
+    aliases: [{
+      type: 'Stage Name',
+      prefix: "The Artist Formally Known As...",
+      forenames: "Prince Rogers",
+      surname: "Prince Rogers Nelson",
+      suffix: "Unpronounceable Symbol"
+    }, {
+      type: 'Undercover',
+      prefix: "Agent",
+      forenames: "Alec",
+      surname: "Trevelyan",
+      suffix: "006"
+    }]
+  },
+  informant: {
+    forenames: "Informant Forename1 Informant Forename2",
+    surname: "Informant Surname",
+    address: "Informant Address",
+    qualification: "Confidential Informant",
+    signature: "Here is a signature"
+  },
+  registrar: {
+    signature: "Registrar signature",
+    designation: "Registrar",
+    subdistrict: "Subdistrict town",
+    district: "District city",
+    administrativeArea: "Adminshire"
+  },
+  coroner: {
+    name: "Informy McInformface",
+    designation: "Coroner",
+    area: "Adminshire"
+  },
+  partner: {
+    name: "Partner name",
+    occupation: "Partner occupation",
+    retired: true
+  },
+  father: {
+    name: "Father name",
+    occupation: "Father Occupation"
+  },
+  mother: {
+    name: "Mother name",
+    occupation: "Mother occupation"
+  },
+  status: {
+    blocked: false,
+    correction: "None",
+    marginalNote: "None",
+    onAuthorityOfRegistrarGeneral: false
+  },
+  previousRegistration: {},
+  nextRegistration: {}
```

Comment:
I don't think these are valid values.
Better to set them to `undefined` I think.